src/builder/modules/Generator_Builder.res

```diff
@@ -437,6 +437,48 @@ let fillBackgroundColor = (model: Model.t, fillStyle: string) => {
   }
 }
 
+let drawLine = (
+  model: Model.t,
+  (dx, dy, dw, dh): rectangle,
```

Comment:
Let's use the `position` type up above rather than a rectangle. Lines are typically represented by two points (x1, y1) and (x2, y2) rather than a rectangle (x, y, w, h). E.g.

```
from: position,
to: position,
```